ublox/src/ubx_packets.rs

```diff
@@ -14,9 +14,28 @@ pub trait UbxPacketMeta {
     const MAX_PAYLOAD_LEN: u16;
 }
 
-pub(crate) const SYNC_CHAR_1: u8 = 0xb5;
-pub(crate) const SYNC_CHAR_2: u8 = 0x62;
-pub(crate) const RTCM_SYNC_CHAR: u8 = 0xd3;
+/// Constants used when parsing UBX protocol.
+pub(crate) struct UbxConstants;
+
+impl UbxConstants {
+    /// Size of the UBX header frame markers (in bytes)
+    pub const SYNC_SIZE: usize = 2;
+
+    /// Size of the Payload length encoding (in bytes)
+    pub const PAYLOAD_ENCODING_SIZE: usize = 2;
+
+    /// Total size of the UBX header (SYNC +CLASS +ID +LENGTH)
```

Comment:
Should we add a `pub const CLASS_ID_SIZE: 2`
I see this already pops up in two places, here and in `parser.rs`

```
619: // Payload length identification
           let offset = UbxConstants::SYNC_SIZE + 2; // CLASS + ID
```

Reply:
I will get back to this tomorrow

Reply:
Should it be `pub` or `pub(crate)`?